internal/sshserver/serverImpl.go

```diff
@@ -15,6 +15,7 @@ import (
 	messageCodes "github.com/containerssh/libcontainerssh/message"
 	"github.com/containerssh/libcontainerssh/service"
 	"golang.org/x/crypto/ssh"
+	"github.com/pires/go-proxyproto"
```

Comment:
This is a third party library and should be abstracted away with its own package, including tests so it can be replaced if need be.